### PR TITLE
Miscellaneous small dashboard bug fixes

### LINKED ISSUE #268: Router summary only displays stats of 1st client
The router summary isn't summing over all the client stats of a router.

Example (summary only says 7 requests, should be 7 + 84 + ... ):

screen shot 2016-04-11 at 3 10 00 pm

### PR DESCRIPTION
**Pull request by rmars**
Fixes #268
Also helps with #265

### CODE REVIEW

**Comment by esbie** [comment]
:+1: lgtm thanks for splitting into separate commits!

**Comment by olix0r** [comment]
⭐

File: admin/src/main/resources/io/buoyant/admin/js/router_summary.js [hunk 1]
```diff
@@ -14,11 +14,12 @@ var RouterSummary = (function() {
...
 
   function processResponse(data, routerName, metricName) {
-    var datum = Query.find(Query.clientQuery().withRouter(routerName).withMetric(metricName).build(), data);
-    return !datum ? 0 : datum.delta;
+    var datum = Query.filter(Query.serverQuery().allServers().withRouter(routerName).withMetric(metricName).build(), data);
+    return _.sumBy(datum, "delta");
   }
 
   function getSuccessAndFailureRate(result) {
+    if (_.isUndefined(result.failures)) result.failures = null;
```
Comment (rmars) [inline admin/src/main/resources/io/buoyant/admin/js/router_summary.js]: Oliver's branch (#258) adds `|| 0` to these arguments, so I think I'll remove this line

Comment (rmars) [inline admin/src/main/resources/io/buoyant/admin/js/router_summary.js]: Oh, the failure rate calculation also uses it, so I will just leave as is.

**Pull request merged**